app/controllers/teaching_classes_controller.rb

```diff
@@ -68,6 +70,18 @@ def find_teaching_class
     render_failed_response("Record not found for ID #{params[:id]}")
   end
 
+  def assign_subjects_to_class
+    ids = params['teaching_class']['subject_ids'].split(',').map(&:to_i)
+    subjects = Subject.find(ids)
+    @teaching_class.subjects << subjects
+  end
+
+  def remove_subjects_from_class
```

Comment:
@josephelias-rubyians  This should be delete method of API for intermediate table for class and subjects